.travis.yml

```diff
@@ -16,23 +16,37 @@ before_install: git submodule update --init
 
 env:
     global:
+        - ARTEFACTS_DIR=build/travis-artefacts
```

Comment:
I think it is called "artifact" though ;)

Reply:
Not in UK English spelling AFAIK. @gavin-norman-sociomantic ?

Reply:
Apparently, yes: http://grammarist.com/spelling/artefact-artifact/

(I've actually always wondered what the difference is, if any.)

Reply:
> The two forms vied for ascendancy in British writing through much of the 20th century until artefact finally gained the upper hand around 1990

>_<

Reply:
OK, changed to artifacts as it seems to be the most popular spelling according to google search.